sdks/universal-router-sdk/src/entities/actions/unwrapWETH.ts

```diff
@@ -7,7 +7,7 @@ import { ROUTER_AS_RECIPIENT, WETH_ADDRESS } from '../../utils/constants'
 
 export class UnwrapWETH implements Command {
   readonly tradeType: RouterActionType = RouterActionType.UnwrapWETH
-  readonly permit2Data: Permit2Permit
+  readonly permit2Data?: Permit2Permit
```

Comment:
intentional change?

Reply:
yes, same as last PR. it fixes a typescript issue